homeassistant/components/integration/sensor.py

```diff
@@ -37,11 +43,13 @@
 from homeassistant.helpers.entity_platform import AddEntitiesCallback
 from homeassistant.helpers.event import (
     EventStateChangedData,
+    async_call_later,
     async_track_state_change_event,
 )
 from homeassistant.helpers.typing import ConfigType, DiscoveryInfoType
 
 from .const import (
+    CONF_MAX_AGE,
```

Comment:
Agree, `max_age` was not my idea and does not have my preference, but it was suggested by reviewers. https://github.com/home-assistant/core/pull/110685#issuecomment-1952873339  

Initially it was named `max_dt` and I meant the max time [differential](https://en.wikipedia.org/wiki/Differential_(mathematics)) in mathematical integration. This is common notation. What do you think about this name?

Less technical options `max_time_differential` or `max_time_period`? Suitable imo because 
- it sets an upper bound on the time width of a Riemann rectangle
- implies that it's about a period

Open to suggestions here.

Reply:
Looking at https://en.wikipedia.org/wiki/Riemann_integral I see those intervals are named _sub-interval_ and the config defines a max on those, so I named it `max_sub_interval`. 

The config is now close to theory about Riemann integration and even without the theory I think for most users it will be clear that it sets a duration, especially with the duration selector.